src/main/kotlin/no/nav/syfo/api/Clients.kt

```diff
@@ -1,7 +1,8 @@
 package no.nav.syfo.api
 
-data class ClientsEnvironment(
-    val istilgangskontroll: ClientEnvironment
+data class Clients(
```

Comment:
Trenger man å endre dette i denne commiten? 🤔

Reply:
Det er en så liten endring så hvorfor ikke tenker jeg 😊 Men vi squasher vel alle commits når vi merger uansett så blir vel ikke mulig å se hvilken spesifikk commit den tilhører? 🤔

Reply:
Pros and cons begger veier der. Det blir tydeligere med egen PR. Vi gjør det også mer tungvint å gjøre små kontinuerlige forbedringer. For meg har jeg merket at det blir vanskeligere og fort gjort å glemme om vi skal lage en ny PR for hver renaming vi gjør.

Reply:
Fjernet, så får vi heller ta det på et senere tidspunkt 😊